saleor/graphql/core/mutations.py

```diff
@@ -157,6 +158,7 @@ def __init_subclass_with_meta__(
         cls,
         auto_permission_message=True,
         description=None,
+        doc_category=None,
```

Comment:
Wouldn't be good to extend the `ModelMutation` to automatically assign the correct `doc_category` based on model that we provided? This should simplify the creation of the mutation and the amount of work to update all existing ModelMutations. The input `doc_category` could be there and if not provided we could use map.
Something like this one:
```
{
   "User": DOC_CATEGORY_USERS,
   "Group": DOC_CATEGORY_USERS
   "Product": DOC_CATEGORY_PRODUCTS
}

Reply:
I'll try to add something like that; it should help with mutations. Maybe something similar could be used for model-based types. For many other types, it will have to be added manually anyway.